user/sonarcloud.md

```diff
@@ -76,23 +76,14 @@ SonarCloud can inspect internal pull requests of your repository and write comme
 
 > For security reasons, this advanced feature works only for **internal** pull requests. In other words, pull requests built from forks won't be inspected.
 
-To activate analysis on pull requests, you need to follow those extra steps:
+To activate analysis on pull requests, you simply need to [install the SonarCloud application](https://github.com/apps/sonarcloud) on your GitHub organization(s).
 
-1. Generate a [personal access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) for the GitHub user which will be used by SonarCloud to write the comments.
-  - This GitHub user should not be one of the developers, but rather a technical account which has write access to the repository and which will act as a bot
-  - The token must have the following scopes:
-    -  "repo:status" and "public_repo" for public repositories
-    -  all of "repo" scope for private repositories
-2. Go to the "Administration > General Settings > Pull Requests" page of your project on SonarCloud
-  - Enter this token in the "GitHub > Authentication token" section 
-
-> When specifying the token in SonarCloud, make sure that you click twice on "Save"! To be sure that your token was saved, reload the administration page and make sure that you see a "Change" button on the "Authentication token" section.
+Note that if you are a long-time SonarCloud user and you previously configured GitHub tokens on your projects, you should now delete those tokens from the "Administration > General Settings > Pull Requests" page of your projects.
```

Comment:
And we should maybe remove "long-time SonarCloud user", and just leave it as "if you used SonarCloud before the GitHub application" or something? What do you think?

Reply:
Feedback applied, thanks!